src/components/WineList.jsx

```diff
@@ -20,13 +20,13 @@ const DataCepage = () => {
   return (
     <>
       {data.map((o) => (
-        <div key={o} className="card">
+        <section key={o} className="card">
           <h2 className="card-title">{o}</h2>
-          <img className="card-img" src={Glasses} alt="verre" />
-        </div>
+          <img className="card-img" src={Glasses} alt="à déguster" />
```

Comment:
à terme, cette page contiendra des liens/boutons : le alt doit contenir du texte qui explique le but du lien/bouton

https://html.spec.whatwg.org/#a-link-or-button-containing-nothing-but-the-image

le texte sera sûrement à changer une fois le vin dégusté :)